### PR TITLE
Fixed some typos and grammatical errors

### LINKED ISSUE #18307: [translation]: Update Spanish translations for contributing documentations
#### Update the following in Spanish from their English version available in [`/docs`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs) directory:

- [ ] [`/docs/spanish/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/CONTRIBUTING.md) from its english version at [`/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/CONTRIBUTING.md)

- [ ] [`/docs/spanish/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/README.md) from its english version at [`/docs/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/README.md)

- [ ] [`/docs/spanish/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/how-to-catch-outgoing-emails-locally.md) from its english version at [`/docs/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-catch-outgoing-emails-locally.md)

- [ ] [`/docs/spanish/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/how-to-setup-freecodecamp-locally.md) from its english version at [`/docs/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-setup-freecodecamp-locally.md)

- [ ] [`/docs/spanish/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/how-to-work-on-coding-challenges.md) from its english version at [`/docs/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-coding-challenges.md)

- [ ] [`/docs/spanish/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/spanish/how-to-work-on-guide-articles.md) from its english version at [`/docs/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-guide-articles.md)
 
#### Once you are ready please make a pull request, preferably one for each file.

When you make a PR for this particular issue, do not tag it with `Closes #XXXXX` instead just reference it with `#XXXXX`

### PR DESCRIPTION
**Pull request by Catherinesjkim**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #18307

### CODE REVIEW

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
-    (Opcional): Recomendamos haer un mensaje convencional. Esta es una buena práctica que verás en algunos de los repositorios Open Source más populares. Como desarrollador, deberías seguir las prácticas estándar.
+    (Opcional): Recomendamos hacer un mensaje convencional. Esta es una buena práctica que verás en algunos de los repositorios Open Source más populares. Como desarrollador, deberías seguir las prácticas estándar.
 
     Algunos ejemplos de mensajes convencionales serían:
 
     ```md
     Solución: Actualizar articúlo de guía sobre HTML
     Solución: Actualizar scripts para Travis-CI
-    feat: Añadir articulo sobre JavaScript hoisting
+    feat: Añadir articulo sobre JavaScript hosting
```
Comment (tbushman) [inline docs/spanish/how-to-work-on-guide-articles.md]: Whoever reviews this for translation -- this should remain as `hoisting`

**Review by tbushman — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@tbushman What is the status with this PR?

**Comment by tbushman** [comment]
@RandellDawson Needs Spanish proofreading.

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
 
 ## Pasos para crear y editar artículos de la Guía
 
 1. 🍴 [Fork este repositorio](https://github.com/freeCodeCamp/freeCodeCamp#fork-destination-box)
-2. 👀️ Sgiue las normas de controbución expuestas a continuación.
-3. 🔧 Propón cambios asombrosos!
-4. 📖 Lee la [guía de buenas prácticas de estilo](/docs/style-guide-for-guide-articles).
-5. 👉 [Haz una Pull Request](https://github.com/freeCodeCamp/freeCodeCamp/compare)
+2. 👀️ Suigue las normas de contribución expuestas a continuación.
+3. 🔧 Propónga cambios asombrosos!
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: Most of this article is written using the second person informal (tú) grammatical person. I would continue using this to be consistent.

So, we should keep ***Propón***

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+4. 📖 Lea la [guía de buenas prácticas de estilo](/docs/style-guide-for-guide-articles).
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: Keep ***Lee***

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
 
 - [Utilizando la interfaz de GitHub en tu navegador](#using-the-github-web-interface-on-your-browser).
-- [Trabajando en tu ordenador personal](#working-on-your-local-machine) (_recomendado_ para previsualiar cambios).
+- [Trabajando en tu ordenador personal](#working-on-your-local-machine) (_recomendado_ para pre-visualizar cambios).
 
-#### Utilizar la interaz web de GitHub
+#### Utilizar la interfaz web de GitHub
 
-Mira este vídeo de demostración o sigue los siguientes pasos:
+Mira este video de demostración o sigue los siguientes pasos:
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: Mira este ***vídeo***

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 5]
```diff
@@ -1,51 +1,51 @@
...
 
 **[TODO]** Update the GIF recording.
 
 ![GIF showing the GitHub interface steps](#)
 
-1. Ve a la carpets **"páginas"** (situado en [`guide`](/guide)) donde encontrarás el artículo raiz que quieras editar.
+1. Vea las carpetas **"páginas"** (situado en [`guide`](/guide)) donde encontrarás el artículo raiz que quieras editar.
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: ***Ve***

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 6]
```diff
@@ -1,51 +1,51 @@
...
 
     > Todas las raíces estarán en un archivo index.md
 
-2. Pincha en <kbd>Editar este archivo</kbd> y haz tus cambios al archivo en la consola de edición de GitHub.
+2. Pincha en <kbd>Editar este archivo</kbd> y hace tus cambios al archivo en la consola de edición de GitHub.
 
-    > Si el icono aparece gris y te muestra la alerta "Debes estar en una rama para hacer o proponer cambios a este archivo", significa que probablemente estés en la rama de otra persona. En la parte superior izquierda de la página hay una casilla desplegable que dice: "Árbol: #######". Pincha en el desplegable y cambia la rama a maestra. El icono de edición debería estar disponible ahora.
+    > Si el icono aparece gris y te muestra la alerta "Debes estar en una rama para hacer o proponer cambios a este archivo", significa que probablemente estas en la rama de otra persona. En la parte superior izquierda de la página hay una casilla desplegable que dice: "Árbol: #######". Pincha en el desplegable y cambia la rama a maestra. El icono de edición debería estar disponible ahora.
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: probablemente ***estás*** en la rama

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 7]
```diff
@@ -56,7 +56,7 @@ Mira este vídeo de demostración o sigue los siguientes pasos:
     Documentos: Actualizadas recomendaciones de contribución
     ```
 
-    Se breve, no más de 50 caracteres. Puedes añadir información extra en la descripción del mensaje.
+    Sea breve, no más de 50 caracteres. Puedes añadir información extra en la descripción del mensaje.
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: ***Sé***

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 8]
```diff
@@ -66,17 +66,17 @@ Mira este vídeo de demostración o sigue los siguientes pasos:
 
 5. En la siguiente pantalla, puedes añadir más detalles sobre tu PR, luego pincha en <kbd>Crear pull request</kbd>.
 
-Enhorabuena 🎉! Acabas de crear una pull request.
+Felicitaciones 🎉! Acabas de crear una pull request.
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: ***un*** pull request

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 9]
```diff
@@ -66,17 +66,17 @@ Mira este vídeo de demostración o sigue los siguientes pasos:
...
 
 #### Trabajar desde tu sistema local (_recomendado_ para revisar cambios)
 
-No es obligatorio que trabajes en tu sistema personal, salvo que desees previsualizar tus cambios, o trabajar con mejoras y arreglos de UI. También es recomendable si tienes problemas con git como errores de integración, rebase, etc.
+No es obligatorio que trabajes en tu sistema personal, salvo que desees pre-visualizar tus cambios, o trabajar con mejoras y arreglos de UI. También es recomendable si tienes problemas con git como errores de integración, rebase, etc.
 
-##### Lee sobre esta recomendaciones en [Cómo configurar freeCodeCamp localmente](/docs/how-to-setup-freecodecamp-locally.md)
+##### Lea sobre estas recomendaciones en [Cómo configurar freeCodeCamp localmente](/docs/how-to-setup-freecodecamp-locally.md)
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: ***Lee*** sobre estas recomendaciones

File: docs/spanish/how-to-work-on-guide-articles.md [hunk 10]
```diff
@@ -109,7 +109,7 @@ Los revisores harán todos los esfuerzos posibles para resolver estos conflictos
 
 #### Solicitar cambios
 
-Si la Pull Request no es perfect el revisor podría:
+Si la Pull Request no es perfecta el revisor podría:
```
Comment (yvette-chica) [inline docs/spanish/how-to-work-on-guide-articles.md]: Si ***el*** Pull Request

**Review by yvette-chica — CHANGES_REQUESTED** [review]
Good job correcting many errors! I noticed you changed a few second person informal (tú) imperative conjugations to second person formal (usted). I left comments where this happened.

**Comment by tomaspozo** [comment]
This is a very old PR and there are several typos which are fixed on the latest version of the file.

**Pull request closed without merging**